flink-runtime/src/test/java/org/apache/flink/runtime/io/disk/iomanager/AsynchronousFileIOChannelTest.java

```diff
@@ -47,7 +45,6 @@
 import static org.junit.Assert.fail;
 import static org.mockito.Mockito.mock;
```

Comment:
nit: How about removing the remaining mocks of this class as well, It shouldn't be complicated either.

Reply:
While you can easily mock `WriteRequest` a `Buffer` is an entirely different beast.

Reply:
My idea is that the tests in the `AsynchronousFileIOChannelTest` class should not care about the specific implementation of the buffer, so we can simply construct a real buffer instead of mocking it(maybe via `BufferBuilderTestUtils.buildSomeBuffer();`). 

But, It's fair enough not to make this change as we have extensively used mocks in the `runtime` module and cannot completely remove them at this time.

Reply:
The BufferBuilderTestUtils should work. wasnt aware that we had something like that :+1: